uat/test-iot-device-sdk-mqtt5-client/.gitignore

```diff
@@ -0,0 +1 @@
+/target/
```

Comment:
please just put everything under `uat`, not an additional directory `test-iot-device....`

Reply:
Fixed in d73e03879e39a7be0dc17306d2b33f6da9cff331